app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,12 @@
   </head>
 
   <body>
-    <%= yield %>
+    <% if notice.present? %>
+      <p class="alert alert-info"><%= notice %></p>
+      <% end %>
+      <% if alert.present? %>
+      <p class="alert alert-danger"><%= alert %></p>
+      <% end %>
+      <%= yield %>
```

Comment:
You'll probably want to fix the tabbing here. The code is fine, but the whitespace should look more like this:

```
 +    <% if notice.present? %>
 +      <p class="alert alert-info"><%= notice %></p>
 +    <% end %>
 +    <% if alert.present? %>
 +      <p class="alert alert-danger"><%= alert %></p>
 +    <% end %>
 +    <%= yield %>
```